### ISSUE TITLE
[Menu design][Accessibility] Close button is labeled incorrectly

### ISSUE THREAD

**Issue by natastro** [mozilla-mobile/firefox-ios#23492]
## Prerequisites

- Turn on VoiceOver
- Turn on treatment-a of the iOS Menu Redesign
- Found on Firefox Beta iOS app 134.0
- Open any bookmarked webpage

## Steps to reproduce

1. Open Firefox Beta iOS app
2. Open the menu from the hamburger menu icon button at the bottom of the screen
3. As the action sheet appears on screen, navigate to the Save submenu and double tap to activate
4. Navigate to 'Edit bookmark', then double tap to activate
5. Move VoiceOver focus to the Close 'x' button at the top left of the sheet. Observe VoiceOver output

### Expected behavior / Actual behavior

VoiceOver output for the Close 'x' button says: _'cross large, button, close'_
The expected VoiceOver output should be '_close, button_'

### Device & build information
* Device: iPhone 15, iOS 18.1.1
* Build version: Firefox Beta 134.0
* First seen version: First time I've seen this issue
### Notes
Attachments: 

https://github.com/user-attachments/assets/a504d063-e15c-4433-95c7-31a62f2b08e3

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-10769)

**Comment by natastro** [comment]
Attaching screenshot of screen when using Voice Control. Names overlay shows the 'CrossLarge' label as well

![IMG_5044](https://github.com/user-attachments/assets/f4c8a990-f652-4122-9a9f-2ab51dbc7110)
